bp_top/test/tb/bp_tethered/bsg_tag_boot.tr

```diff
@@ -0,0 +1,247 @@
+# | _ \ (_) | (_) || | \__ \ | | |   / / _ \|  _/ | (__| |__| ' < (_ | _|| .` |
+# |___/\___/ \___/ |_| |___/ |_| |_|_\/_/ \_\_|    \___|____|_|\_\___|___|_|\_|
+#
+################################################################################
+
+################################################################################
+#
+# RESET BSG TAG CLIENTS
+#
+# Next, we should reset each client node. To do this we send a packet
+# that has all 1's for data, and has data_not_reset=0. The nodeID should
+# be the ID of the client we are reseting, and length should be the
+# corrent length of the packet. We should send this packet to each client.
+
+#SEND  en  id=0   r l=1
+0001___1___000000_0_0001___000000001
+#SEND  en  id=1   r l=5
+0001___1___000001_0_0101___000011111
+#SEND  en  id=2   r l=5
+0001___1___000010_0_0101___000011111
+#SEND  en  id=3   r l=5
+0001___1___000011_0_0101___000011111
+#SEND  en  id=4   r l=5
+0001___1___000100_0_0101___000011111
+#SEND  en  id=5   r l=1
+0001___1___000101_0_0001___000000001
+#SEND  en  id=6   r l=1
+0001___1___000110_0_0001___000000001
+#SEND  en  id=7   r l=1
+0001___1___000111_0_0001___000000001
+#SEND  en  id=8   r l=1
+0001___1___001000_0_0001___000000001
+#SEND  en  id=9   r l=3
+0001___1___001001_0_0011___000000111
+
+### Set delay line triggers low
+
+#SEND  en  id=5   d l=1    {trigger}
+0001___1___000101_1_0001___000000000
+#SEND  en  id=6   d l=1    {trigger}
+0001___1___000110_1_0001___000000000
+#SEND  en  id=7   d l=1    {trigger}
+0001___1___000111_1_0001___000000000
+#SEND  en  id=8   d l=1    {trigger}
+0001___1___001000_1_0001___000000000
+
+### Program the delay lines phase shift
+
+#SEND  en  id=1   d l=5    {adt, cdt, fdt}
+0001___1___000001_1_0101___000000000
+#SEND  en  id=2   d l=5    {adt, cdt, fdt}
+0001___1___000010_1_0101___000000000
+#SEND  en  id=3   d l=5    {adt, cdt, fdt}
+0001___1___000011_1_0101___000000000
+#SEND  en  id=4   d l=5    {adt, cdt, fdt}
+0001___1___000100_1_0101___000000000
+
+### Trigger delay lines
+
+#SEND  en  id=5   d l=1    {trigger}
+0001___1___000101_1_0001___000000001
+0001___1___000101_1_0001___000000000
+#SEND  en  id=6   d l=1    {trigger}
+0001___1___000110_1_0001___000000001
+0001___1___000110_1_0001___000000000
+#SEND  en  id=7   d l=1    {trigger}
+0001___1___000111_1_0001___000000001
+0001___1___000111_1_0001___000000000
+#SEND  en  id=8   d l=1    {trigger}
+0001___1___001000_1_0001___000000001
+0001___1___001000_1_0001___000000000
+
+### Async clk-gen reset to get things moving
+
+#SEND  en  id=0   d l=1    {async_reset}
+0001___1___000000_1_0001___000000000
+0001___1___000000_1_0001___000000001
+0001___1___000000_1_0001___000000000
+
+### Set downsamples and reset
+#
+#SEND  en  id=9   d l=2    {ds_val, reset}
+0001___1___001001_1_0011___000000001
+0001___1___001001_1_0011___000000000
+
+################################################################################
```

Comment:
Nit, but this isn't I/O complex anymore, is it?

Reply:
Yeah I forgot to remove that comment line.